Removed deprecated functions, widgets and solved some issues

manoj1749 opened the pull request:
closes #177
closes #184
closes #185 

- This pr will be resolving the issues 
    - #177 
    - #184 
    - #185 
- Had formatted the code and replaced the deprecated FlatButton with a suitable widget and had solved the errors and issues found when we analyzed the prioject's dart code and when we ran flutter unit tests for the project

manoj1749 committed:
replaced FlatButton with TextButton

manoj1749 committed:
resolved deprecated toggleableActiveColor

manoj1749 committed:
primary method for Elevatedbutton is deprecated

manoj1749 committed:
renamed subtitle1 to titleMedium as subtitle1 is deprecated

manoj1749 committed:
fixed error with `flutter pub get` during workflow

fixed error with `flutter pub get` during workflow

updated modal_bottom_sheet version due to issues with flutter

formatted file

updated packages and sdk version

updated packages

changed modal_bottom_sheet version

Revert "changed modal_bottom_sheet version"

This reverts commit 89030b19553082dd78502b812a1618b6ecb5c564.

Revert "updated packages"

This reverts commit f255089fdc1692e2a7efa9cdb764a71554e305a6.

Revert "updated packages and sdk version"

This reverts commit 94fa9e6997ba6350421b452630980d602de4416d.

Update pubspec.yaml

Update pubspec.yaml

buildwithmalik commented:
@manoj1749 Please make sure the PR passes all checks. Also #185 will be dealt with, It was due to the workflow not being activated for this branch.

manoj1749 commented:
> @manoj1749 Please make sure the PR passes all checks. Also #185 will be dealt with, It was due to the workflow not being activated for this branch.

Ok sure

manoj1749 committed:
fixed workflow errors

updated modal_bottom_sheet

updated flutter-version in workflow file

Update pubspec.lock

Revert "Update pubspec.lock"

This reverts commit 7fff547e0c13d24a077ce3ecdf3d85c81d926ece.

updated file package

manoj1749 commented:
> @manoj1749 Please make sure the PR passes all checks. Also #185 will be dealt with, It was due to the workflow not being activated for this branch.

I had made the necessary changes for the errors encountered when we ran the workflow checks, Now you can apporve for the workflow checks for this pr

manoj1749 commented:
Heyy @AbdulMalikDev, this pr is passing all check, can you look into this once
